features/bevragen/persoon/adressering/adres-regels/fields-alias.feature

```diff
@@ -0,0 +1,149 @@
+
+  Abstract Scenario: consumer is geautoriseerd voor 'adressering buitenland' en vraagt het <field> veld van de adressering gegevensgroep
+    Gegeven de consumer is geautoriseerd voor 'adressering binnenland' gegevens
+    En de consumer is geautoriseerd voor 'adressering buitenland' gegevens
+    En de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | naam                             | waarde                      |
+    | land adres buitenland (13.10)    | 6014                        |
+    | regel 1 adres buitenland (13.30) | 1600 Pennsylvania Avenue NW |
+    | regel 2 adres buitenland (13.40) | Washington, DC 20500        |
+    | regel 3 adres buitenland (13.50) | Selangor                    |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000097                       |
+    | fields              | adressering.<field>             |
+    Dan heeft de response een persoon met de volgende 'adressering' gegevens
+    | naam    | waarde   |
+    | <field> | <waarde> |
+
+    Voorbeelden:
+    | field       | waarde                      |
+    | adresregel1 | 1600 Pennsylvania Avenue NW |
+    | adresregel2 | Washington, DC 20500        |
+    | adresregel3 | Selangor                    |
+
+  Scenario: consumer is geautoriseerd voor 'adressering buitenland' en vraagt het land veld van de adressering gegevensgroep
+    Gegeven de consumer is geautoriseerd voor 'adressering binnenland' gegevens
+    En de consumer is geautoriseerd voor 'adressering buitenland' gegevens
+    En de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | naam                             | waarde                      |
+    | land adres buitenland (13.10)    | 6014                        |
+    | regel 1 adres buitenland (13.30) | 1600 Pennsylvania Avenue NW |
+    | regel 2 adres buitenland (13.40) | Washington, DC 20500        |
+    | regel 3 adres buitenland (13.50) | Selangor                    |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000097                       |
+    | fields              | adressering.land                |
+    Dan heeft de response een persoon met de volgende 'adressering' gegevens
+    | naam              | waarde                       |
+    | land.code         | 6014                         |
+    | land.omschrijving | Verenigde Staten van Amerika |
+
+Rule: de 'adresseringBinnenland' field alias moet worden gebruikt door een consumer niet is geautoriseerd voor het bevragen van adresregels horende bij verblijfplaats buitenland
```

Comment:
'... door een consumer niet is geautoriseerd ...' --> '... door een consumer **die** niet is geautoriseerd ...'